retryrpc/api.go

```diff
@@ -320,10 +332,9 @@ type ClientConfig struct {
 	Callbacks                interface{}   // Structure implementing ClientCallbacks
 	DeadlineIO               time.Duration // How long I/Os on sockets wait even if idle
 	KeepAlivePeriod          time.Duration // How frequently a KEEPALIVE is sent
+	Logger                   *log.Logger   // If nil, defaults to log.New()
```

Comment:
I take it this "eases" integration:
1) Unmodified clients will get the "zero value" nil for this field automatically and, thus,
2) Use the default log.New() logger

I guess this is a nice way to change things w/out having to fix the clients... but each of those clients should be fixed, right?

I'm not worried about them... but I'll certainly be changing iclient (and, similarly, imgr) to supply an appropriate Logger: value.